services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/attribute/ListAttributeConverter.java

```diff
@@ -45,28 +45,36 @@
  *
  * <p>
  * A builder is exposed to allow defining how the collection and element types are created and converted:
+ * <p>
  * <code>
- * AttributeConverter<List<Integer>> listConverter =
+ * {@literal AttributeConverter<List<Integer>> listConverter =
  * CollectionAttributeConverter.builder(EnhancedType.listOf(Integer.class))
  * .collectionConstructor(ArrayList::new)
  * .elementConverter(IntegerAttributeConverter.create())
- * .build()
+ * .build()}
  * </code>
  *
  * <p>
  * For frequently-used types, static methods are exposed to reduce the amount of boilerplate involved in creation:
+ * <p>
  * <code>
- * AttributeConverter<List<Integer>> listConverter =
- * CollectionAttributeConverter.listConverter(IntegerAttributeConverter.create());
+ * {@literal AttributeConverter<List<Integer>> listConverter =
+ * CollectionAttributeConverter.listConverter(IntegerAttributeConverter.create());}
+ * </code>
  * <p>
- * AttributeConverter<Collection<Integer>> collectionConverer =
- * CollectionAttributeConverter.collectionConverter(IntegerAttributeConverter.create());
+ * <code>
+ * {@literal AttributeConverter<Collection<Integer>> collectionConverer =
+ * CollectionAttributeConverter.collectionConverter(IntegerAttributeConverter.create());}
+ * </code>
  * <p>
- * AttributeConverter<Set<Integer>> setConverter =
- * CollectionAttributeConverter.setConverter(IntegerAttributeConverter.create());
+ * <code>
+ * {@literal AttributeConverter<Set<Integer>> setConverter =
+ * CollectionAttributeConverter.setConverter(IntegerAttributeConverter.create());}
+ * </code>
  * <p>
- * AttributeConverter<SortedSet<Integer>> sortedSetConverter =
- * CollectionAttributeConverter.sortedSetConverter(IntegerAttributeConverter.create());
+ * <code>
+ * {@literal AttributeConverter<SortedSet<Integer>> sortedSetConverter =
+ * CollectionAttributeConverter.sortedSetConverter(IntegerAttributeConverter.create());}
  * </code>
```

Comment:
What's the difference between <code> and <pre>?

Reply:
```<code>``` makes the font different, but it cannot guarantee the formats, and ```<pre>``` can let the format of the code be showed as what they are in the original code file, i.e showing the indents and spaces.